lib/msf/base/sessions/command_shell.rb

```diff
@@ -8,6 +8,11 @@
 module Msf
 module Sessions
 
+  class << self
```

Comment:
Can probably be removed as superfluous: https://github.com/rapid7/metasploit-framework/pull/11903#discussion_r305939293